tests/test_watson.py

```diff
@@ -795,6 +795,33 @@ def test_report(watson):
         watson.report(arrow.now(), arrow.now(), tags=["A"], ignore_tags=["A"])
 
 
+def test_report_current(config_dir):
+    watson = Watson(
+        current={'project': 'foo', 'start': arrow.now().shift(hours=-1)},
+        config_dir=config_dir
+    )
+
+    _ = watson.report(
+        arrow.now(), arrow.now(), current=True, projects=['foo']
+    )
+    report = watson.report(
+        arrow.now(), arrow.now(), current=True, projects=['foo']
+    )
+    assert len(report['projects']) == 1
+    assert report['projects'][0]['name'] == 'foo'
+    assert report['projects'][0]['time'] == pytest.approx(3600, rel=1e-2)
```

Comment:
I don't think you need to use `pytest.approx()` as there are no floating-point numbers involved. In fact, asserting that time equals `3600` is equivalent to what you did, as we are working with a granularity of seconds (`int`s). This would lead to a flaky test: if there was a change in seconds between building the `Watson` object and calling the second `watson.report()`, the assertion would fail.

A possible solution could be to assert that time is _less than_ `3600 * 2`, which is enough to test that the time is not duplicated. Any other ideas are more than welcomed! :relaxed: 

Finally, I would use one minute (60) instead of one hour (3600) in order to reduce the complexity of the test (unless there is a reason to use hours that I didn't notice).

It seems we're close to ending this PR :tada:

Reply:
`report['projects'][0]['time']` is (like `timedelta.total_seconds()`) floating-point-valued with microsecond precision, and it usually takes a few milliseconds until the second report is generated.

The flakyness should be just as negligible at the current 36 seconds as it is at 3600 seconds tolerance in an automated test, but could be reduced by mocking `arrow.now` and `arrow.utcnow`.

Reply:
> report['projects'][0]['time'] is (like timedelta.total_seconds()) floating-point-valued with microsecond precision, and it usually takes a few milliseconds until the second report is generated.

I didn't realize that, sorry!

I supposed they were seconds, because that's what is printed in the terminal. Nevertheless, it's the `utils.format_timedelta()` function that removes millisecond information: `seconds = int(delta.total_seconds())`. :+1: 

> The flakyness should be just as negligible at the current 36 seconds as it is at 3600 seconds tolerance in an automated test

You're completely right, there's no flakiness :relaxed: